From c83f890fe941032486096dd2f0ddc1ad75fb828e Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Fri, 23 Nov 2018 13:53:24 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3328-box-liantong: Add sleep mode
 configuration

The board is adjusted with pwm regulator for logic and arm voltage.

The regulator sleep mode is notified by the GPIO pin issued by the soc
The pwm regulator has no pin control for sleep switching, and needs to
notify the soc pmu special processing.

Change-Id: Id2c22f26abcdae427a8e30716afbcaab471ea183
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi
index 119663bdbeef..8d3feef1f63b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi
@@ -495,6 +495,12 @@
 &rockchip_suspend {
 	status = "okay";
 	rockchip,virtual-poweroff = <1>;
+	rockchip,sleep-mode-config = <
+		(0
+		|RKPM_SLP_CTR_VOL_PWM0
+		|RKPM_SLP_CTR_VOL_PWM1
+		)
+	>;
 };
 
 &uart0 {
-- 
2.35.3

